easybuild/easyconfigs/s/STAR/STAR-2.7.10b-GCC-12.2.0.eb

```diff
@@ -2,7 +2,7 @@
 # Guy's and St Thomas' NHS Foundation Trust and King's College London
 # Based on STAR-2.7.7a-GCC-10.2.0.eb 
 # uploaded by J. Sassmannshausen
-# modified by Thomas Eylenbosch
+# Updated: Petr Král (INUITS)
```

Comment:
Please restore the previous attribution and append yours to that.

Reply:
It seems Thomas Eylenbosch was a bit faster then me with creating this file. That's why github shows these strange changes in diff. My version was supposed to be a new file. Now I would keep his version of this file.